packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -145,27 +145,27 @@
 }
 
 .dismissInvertedVariant:focus {
-  box-shadow: 0 0 0 2px var(--ColorShadowDismissInverted);
+  box-shadow: 0 0 0 2px var(--DismissInvertedShadowColor);
```

Comment:
why aren't we using the CSS declaration syntax of `BoxShadow` here? `DismissInvertedBoxShadowColor`

Reply:
Updizzled